docs/api/registration-base-url-resource.md

```diff
@@ -307,9 +323,6 @@ packageContent | string  | no       | The URL to the package content (.nupkg)
 published      | string  | no       | A string containing a ISO 8601 timestamp of when the package was published
 registration   | string  | no       | The URL to the registration index
 
-> [!Note]
-> On nuget.org, the `published` value is set to year 1900 when the package is unlisted.
```

Comment:
Should we keep this note? Or is the note incorrect for the registration leaf? Personally, I would rather we are explicit and repeat this note (especially since people may hop around the article without reading previous section)

Reply:
Yeah, I think you're right. It's an oddity we should make really clear.